lib/smart_answer/date_range.rb

```diff
@@ -102,5 +108,13 @@ def leap_dates
     def leap?
       leap_dates.any?
     end
+
+    def weeks_later(weeks)
+      self + weeks * 7
+    end
+
+    def formatted
```

Comment:
Would it make sense to call it `.to_s`?

`.formatted` to me implies that we can apply different formatting, and then I would expect to use `.to_formatted_s(:format)` on it.

Reply:
@chao-xian 👍